src/criteria.ts

```diff
@@ -1,5 +1,7 @@
 import { DataValue, DataValueMap } from './data';
 
+import { ISelectOption, SchemaCategory } from '@hyperproof/integration-sdk';
```

Comment:
@ndhuutai @Hashiphoto On second thought, I'm not sure this is the right approach.

Adding a package dependency for `ISelectOption` doesn't make sense--too costly.

And I'm not convinced that `SchemaCategory` belongs in integration-sdk.

Comments?

Reply:
Let's move the `SchemaCategory` to hypersync-sdk and also remove `schemaCategory` since it's already on type `IHypersync`

Reply:
we can leave the `ISelectOption` back there but we'll need to keep note to make sure they're consistent

Reply:
It is actually on the Hypersync settings but yeah, I'm with you.  Changes in flight.  Will loop you in on those.